curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804ec.md

```diff
@@ -22,8 +22,8 @@ assert(code.match(/\<html.*?\>/) && code.match(/\<\/html\>/));
 Your `html` element should have a `lang` attribute with the value `en`. You may have omitted the attribute/value, or have a typo.
 
 ```js
-const extraSpacesRemoved = code.replace(/\s+/g, ' ');
-assert(extraSpacesRemoved.match(/\<html lang\=("|')([a-z]+)\1\>/));
+const extraSpacesRemoved = code.replace(/\s+/g, " ");
+assert(extraSpacesRemoved.match(/\<html lang\s*\=\s*("|')en\1\s*>/));
```

Comment:
This will make the quotes optional, and make it possible to have extra spaces in front of the `html` but not 0 spaces `<html    lang="en">`, to make it as lenient as possible.

the hint in 29 and the test from 31 to 33 can go, because they are making the step stricter than needed

https://github.com/freeCodeCamp/freeCodeCamp/issues/49562#issuecomment-1452506732

```suggestion
assert(extraSpacesRemoved.match(/\<html\s+lang\s*\=\s*("?|'?)en\1\s*\>/));
```

Reply:
feel free to make this ready for review, when you push the changes.

Reply:
I have made the change you suggested and removed the hint that was in line 29 and the test from from line 31 to 33.